src/problem2.rb

```diff
@@ -0,0 +1,9 @@
+class Problem2
+  def identify_largest_numbers(array, no_elements_identify)
+    unless array.nil?
```

Comment:
Then how do i check if the array is nil? if the function receives a nil parameter is will surely fail. Shouldn't I treat that case in the function?